tests/smoke/desiredstate_test.go

```diff
@@ -94,7 +97,7 @@ func TestDesiredState(t *testing.T) {
 	require.NoError(t, err)
 	tests.RequireOutputContains(t, output, "The installation is out-of-sync, running the upgrade action")
 
-	installation, err := test.ShowInstallation("operator", "mybuns")
+	displayInstallation, err := test.ShowInstallation("operator", "mybuns")
```

Comment:
Did you mean to change this variable name? Subsequent asserts aren't looking at the correct variable (see two lines down).

Reply:
Oops, I fixed it now